spec/models/ticket_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+describe Ticket do
+  let(:ticket) { build(:ticket) }
+  context 'model validations' do
+    before { ticket }
+    it { expect(ticket).to be_valid }
+
+    it 'is invalid without a user' do
+      ticket.user_id = nil
+      ticket.valid?
+      expect(ticket.errors[:user_id]).to include("can't be blank")
+    end
+
+    it 'is invalid without a title' do
+      ticket.title = '   '
```

Comment:
nie potrzeba wypełniać pustymi znakami wystarczy samo użycie ticket.title = '' lub nawet ticket.title = nil, 
to samo przy innych przykładach